xtransmit/metrics_latency.hpp

```diff
@@ -17,22 +17,26 @@ class latency
 	{}
 
 public:
-	/// Submit new sample for jitter update.
-	/// @param [in] sample_time  the timestamp of the sample
-	/// @param [in] current_time current time to compare the timestamp with
-	void submit_sample(const time_point& sample_time, const time_point& current_time);
+	/// Submit new sample for latency (transmission delay) metric update.
+	/// @param [in] timestamp  packet timestamp extracted from the payload (system clock)
+	/// @param [in] arrival_time  the time packet is received by receiver (system clock)
+	void submit_sample(const time_point& timestamp, const time_point& arrival_time);
 
+	// Reset values at the end of the measurement period.
 	void reset();
 
-	/// Get curent jitter value.
-	long long get_latency_min() const { return m_latency_min_us; }
-	long long get_latency_max() const { return m_latency_max_us; }
-	long long get_latency_avg() const { return m_latency_avg_us; }
+	/// Get current latency value, in microseconds. Min and max values are
+	/// minimum and maximum latencies registered during mesurement period.
+	/// Avg is the smoothed delay (the same coefficient of 1/16 as in RFC 3550 is applied)
+	/// which isn't reset at the start of the new measurement period.
+	long long get_latency_min() const { return m_latency_min; }
+	long long get_latency_max() const { return m_latency_max; }
+	long long get_latency_avg() const { return m_latency_avg; }
 
 private:
-	long long m_latency_min_us = std::numeric_limits<long long>::max();
-	long long m_latency_max_us = std::numeric_limits<long long>::min();
-	long long m_latency_avg_us = -1;
+	long long m_latency_min = std::numeric_limits<long long>::max();
+	long long m_latency_max = std::numeric_limits<long long>::min();
+	long long m_latency_avg = -1;
```

Comment:
The `_us` suffix was intended to hint the unit behind is microseconds. Even though it was not exposed via the `get_latency_min()` and other functions. Otherwise, the unit is not clear, might be a millisecond.
I would rather rename `get_latency_max()` to `get_latency_us_max()`.

Reply:
I've made this for a reason. We have exactly the same situation for all the metrics: latency, jitter, and now delay factor. That's why I've added comments right before "get" function that the value is in microseconds.

Does it still make sense to do? If yes, this should be done for all the "get" functions.